scripts/spanner-er-docker.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-docker run --rm --entrypoint '' -v `pwd`:/go/src/github.com/nakatamixi/spanner-er --workdir="/go/src/github.com/nakatamixi/spanner-er" nakatamixi/spanner-er  /bin/sh -c "/bin/spanner-er $*"
+docker run --rm --entrypoint '' -v `pwd`:/go/src/github.com/nktks/spanner-er --workdir="/go/src/github.com/nakatamixi/spanner-er" nakatamixi/spanner-er  /bin/sh -c "/bin/spanner-er $*"
```

Comment:
```suggestion
docker run --rm --entrypoint '' -v `pwd`:/go/src/github.com/nktks/spanner-er --workdir="/go/src/github.com/nktks/spanner-er" nakatamixi/spanner-er  /bin/sh -c "/bin/spanner-er $*"
```
sorry I missed one thing, but after new docker image is pushed, we may need to fix as above.